src/style.css

```diff
@@ -0,0 +1,25 @@
+* {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 13px;
+}
+
+h1 {
+  font-size: 17px;
+  font-weight: bold;

```

Comment:
checklist

Reply:
Could you please elaborate a bit, cause in the task the requirement is: 
The "Universal Declaration of Human Rights" must be a h1 heading, styled as 17px bold. 

Or you wish that I assigned an id for that and added styles to id only?